### PR TITLE
Add note to MNTP article

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Add a note to MNTP article as suggested here: https://github.com/umbraco/UmbracoDocs/issues/5533#issuecomment-2205417259

## Type of suggestion

* [x] New content

## Product & version (if relevant)
Umbraco v10, 12, 13 & 14

### CODE REVIEW

**Comment by erikjanwestendorp** [comment]
@sofietoft Added a little note as suggested. Please let me know if the text is ok, then I can update the other articles as well.

**Comment by eshanrnh** [comment]
Thanks for the PR, @erikjanwestendorp 🙌 We will take a look at it as soon as we can 💪

**Review by eshanrnh — APPROVED** [review]
The text looks good, @erikjanwestendorp 🙌 Please feel free to add the Note in other articles as well.. 👏

File: 10/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/multinode-treepicker.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ Enter `typeAlias,altTypeAlias` to only allow selecting nodes with those alias'.
 
 See the example below to see how a value can be added or changed programmatically. To update a value of a property editor you need the [Content Service](../../../../reference/management/services/contentservice/).
 
+:::note
+The example below demonstrates how to add values programmatically using a Razor view. However, this is used for illustrative purposes only and is not the recommended method for production environments.
+:::
```
Comment (eshanrnh) [inline 10/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/multinode-treepicker.md:78]:
```suggestion
{% hint style="info" %}
The example below demonstrates how to add values programmatically using a Razor view. However, this is used for illustrative purposes only and is not the recommended method for production environments.
{% endhint %}
```

File: 12/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/multinode-treepicker.md [hunk 2]
```diff
@@ -79,6 +79,10 @@ Enter `typeAlias,altTypeAlias` to only allow selecting nodes with those alias'.
 
 See the example below to see how a value can be added or changed programmatically. To update a value of a property editor you need the [Content Service](../../../../reference/management/services/contentservice/).
 
+:::note
+The example below demonstrates how to add values programmatically using a Razor view. However, this is used for illustrative purposes only and is not the recommended method for production environments.
+:::
```
Comment (eshanrnh) [inline 12/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/multinode-treepicker.md:84]:
```suggestion
{% hint style="info" %}
The example below demonstrates how to add values programmatically using a Razor view. However, this is used for illustrative purposes only and is not the recommended method for production environments.
{% endhint %}
```

File: 13/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/multinode-treepicker.md [hunk 3]
```diff
@@ -196,6 +196,10 @@ When opening the picker on the `Umbraco anno MMXXIII` node, it will now show the
 
 See the example below to see how a value can be added or changed programmatically. To update a value of a property editor you need the [Content Service](../../../../reference/management/services/contentservice/).
 
+:::note
+The example below demonstrates how to add values programmatically using a Razor view. However, this is used for illustrative purposes only and is not the recommended method for production environments.
+:::
```
Comment (eshanrnh) [inline 13/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/multinode-treepicker.md:201]:
```suggestion
{% hint style="info" %}
The example below demonstrates how to add values programmatically using a Razor view. However, this is used for illustrative purposes only and is not the recommended method for production environments.
{% endhint %}
```

File: 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/multinode-treepicker.md [hunk 4]
```diff
@@ -198,6 +198,10 @@ When opening the picker on the `Umbraco anno MMXXIII` node, it will now show the
 
 See the example below to see how a value can be added or changed programmatically. To update a value of a property editor you need the [Content Service](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.ContentService.html).
 
+:::note
+The example below demonstrates how to add values programmatically using a Razor view. However, this is used for illustrative purposes only and is not the recommended method for production environments.
+:::
```
Comment (eshanrnh) [inline 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/multinode-treepicker.md:203]:
```suggestion
{% hint style="info" %}
The example below demonstrates how to add values programmatically using a Razor view. However, this is used for illustrative purposes only and is not the recommended method for production environments.
{% endhint %}
```

**Comment by erikjanwestendorp** [comment]
@eshanrnh Thanks for the review, just updated the PR with your suggestion.

**Review by eshanrnh — COMMENTED** [review]
I changed the Note formatting to match Gitbook's markdown format: https://docs.gitbook.com/content-editor/blocks/hint#representation-in-markdown

**Comment by eshanrnh** [comment]
Wow, that was super quick 👏

**Pull request merged**